### PR TITLE
Add Code.loaded?/1

### LINKED ISSUE #12313: Expose Code.loaded? now that erlang:module_loaded has no warnings

### PR DESCRIPTION
**Pull request by v0idpwn**
Closes #12313

### CODE REVIEW

File: lib/elixir/test/elixir/code_test.exs [hunk 1]
```diff
@@ -358,6 +358,11 @@ defmodule CodeTest do
     end
   end
 
+  test "loaded?/1" do
+    assert Code.loaded?(__MODULE__)
+    refute Code.loaded?(Code.NoFile)
+  end
+
```
Comment (v0idpwn) [inline lib/elixir/test/elixir/code_test.exs:365]:
```suggestion
```
This is redundant with the doctests, can be removed if you will.

File: lib/elixir/lib/code.ex [hunk 2]
```diff
@@ -1687,6 +1687,26 @@ defmodule Code do
     end
   end
 
+  @doc """
+  Returns true if the module is loaded.
```
Comment (whatyouhide) [inline lib/elixir/lib/code.ex:1691]:
```suggestion
  Returns `true` if the module is loaded.
```

**Review by whatyouhide — APPROVED** [review]

File: lib/elixir/lib/code.ex [hunk 3]
```diff
@@ -1687,6 +1687,26 @@ defmodule Code do
...
+
+      iex> Code.loaded?(Atom)
+      true
+
+      iex> Code.loaded?(NotYetLoaded)
+      false
+
+  """
+  @doc since: "1.15.0"
+  def loaded?(module) do
```
Comment (whatyouhide) [inline lib/elixir/lib/code.ex:1706]:
```suggestion
  @spec loaded?(module) :: boolean
  def loaded?(module) do
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**